client/modules/core/controllers/events.ctrl.js

```diff
@@ -82,68 +81,11 @@ function EventsCtrl($scope, PostsCluster, SocialMediaPost, Event) {
     $scope.selectedEvents = events;
   }
 
-  $scope.filter = filter;
```

Comment:
Another added benefit of using the events as written - the UI should be more responsive!!!